docs/reference/glossary/high-level-terminology.md

```diff
@@ -0,0 +1,61 @@
+---
+id: high-level-terminology
+title: High Level Terminology
+sidebar_label: High Level Terminology
+---
+
+### Networking Transport Layer
+
+The transport layer collects message segments from applications, and transmits them into the network. The transport layer is also responsible for the management of error correction, providing quality and reliability to the end user.
+
+This layer enables the host to send and receive error corrected data, packets or messages over a network 
+
+![](../../../static/img/networking-transport.png)
+
+### Client Runtime
+
+
+### Server Runtime
+
+
+### Netcode
+
+Is a blanket term for anything that somehow relates to networking in online games; netcode is a term most commonly used by gamers when discussing synchronization issues between clients and servers.
+
+
+
+###  Client 
+Is a network client that connects an individual user to the server, used mainly in multiplayer video games. It collects data such as score, player status, position and movement from a single player and sends it to the  server, which allows the server to collect each individual's data and show every player in game.
+
+###  Server 
+
+Is a server which is the authoritative source of events in a multiplayer video game. The server transmits enough data about its internal state to allow its connected clients to maintain their own accurate version of the game world for display to players. They also receive and process each player's input
+
+
+### Hit Registration
+
+This is how a shooter determines if your shot hit something. 
+
+There are two forms of hit registration.
+
+1. Games where bullet travel time is not a factor in close quarters combat can use a simple and fast formwhere bullets have no travel time. 
```

Comment:
This is heavily simplified. There are not just two cases. The non physics version of bullets via raycast is often also used for weapons with travel time such as snipers. Parabolas of bullets can be calculated without the need of a physics simulation.

Reply:
text updated to soften inference. noted added to show more than two forms